website/docs/07-apps/01-included-in-onion/retroarch.md

```diff
@@ -48,17 +48,17 @@ A full Retroarch in a such tiny device ! Retroarch is regularly updated from the
 
 Running from apps section allows you to start RetroArch without immediately loading a game, useful for changing global settings in RetroArch.
 
-To display the Retroarch Menu press <kbd>Menu</kbd> + <kbd>Select</kbd>.
+To display the RetroArch Menu press <kbd>Menu</kbd> + <kbd>Select</kbd>.
 
-Meny additional shortcuts are available for Retroarch, do not forget to check [Onion in-game shortcuts](shortcuts).
+Meny additional shortcuts are available for RetroArch, do not forget to check [Onion in-game shortcuts](shortcuts).
```

Comment:
Again, not you - Can you amend "Meny" to "Many" while we have the PR up please! :)

Reply:
I missed!